### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by grantwwu**
s/is/it/

### CODE REVIEW

File: site2/docs/concepts-tiered-storage.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ One way to alleviate this cost is to use Tiered Storage. With tiered storage, ol
 
 ![Tiered Storage](assets/pulsar-tiered-storage.png)
 
-> Data written to bookkeeper is replicated to 3 physical machines by default. However, once a segment is sealed in bookkeeper is becomes immutable and can be copied to long term storage. Long term storage can achieve cost savings by using mechanisms such as [Reed-Solomon error correction](https://en.wikipedia.org/wiki/Reed%E2%80%93Solomon_error_correction) to require fewer physical copies of data.
+> Data written to Bookkeeper is replicated to 3 physical machines by default. However, once a segment is sealed in Bookkeeper it becomes immutable and can be copied to long term storage. Long term storage can achieve cost savings by using mechanisms such as [Reed-Solomon error correction](https://en.wikipedia.org/wiki/Reed%E2%80%93Solomon_error_correction) to require fewer physical copies of data.
```
Comment (vzhikserg) [inline site2/docs/concepts-tiered-storage.md]: The official name is BookKeeper https://github.com/apache/bookkeeper. All `bookkeeper` words should be updated in this file to have a consistent naming convention.

Comment (grantwwu) [reply site2/docs/concepts-tiered-storage.md]: Okay, I'll do that today.

Comment (sijie) [reply site2/docs/concepts-tiered-storage.md]: @vzhikserg 

just fyi - both `Bookkeeper` and `BookKeeper` are okay to bookkeeper community. It was discussed before.

**Comment by sijie** [comment]
run cpp tests
run integration tests

**Pull request merged**